eng/src/Program.cs

```diff
@@ -11,7 +13,14 @@
 {
     Solutions = new Solution[] { new DotNetSolution( "Metalama.Samples.sln" ) { CanFormatCode = true, BuildMethod = BuildMethod.Build } },
     Dependencies = new[] { Dependencies.PostSharpEngineering, Dependencies.Metalama },
-    RequiresBranchMerging = true
+    Configurations = Product.DefaultConfigurations
+        .WithValue(
+        BuildConfiguration.Public, new BuildConfigurationInfo(
+            MSBuildName: "Release",
+            PublicPublishers: new Publisher[] {
+                new MergePublisher( Pattern.Create( ".empty" ) )
+            } )
```

Comment:
This is not nice. You (i.e. `MergePublisher`) cannot ask the consumer code to do some hack. This means that the API is badly designed. So `MergePublisher` should create the fake pattern itself. 

Better: the hack should be solved and made unnecessary. You cannot rely on a hack (i.e. on a side effect) because hacks increase coupling between component and decreases reliability. 

So the publishing algorithm and object model should be changed to work without specifying a pattern.

Reply:
Resolved by the latest commit in Engineering PR:
https://github.com/postsharp/PostSharp.Engineering/pull/33